src/pages/_app.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
 
-import Footer from '~components/Footer';
-import ToggleColorMode from '~components/ToggleColorMode';
 import theme from '~theme';
 
 import { ChakraProvider } from '@chakra-ui/react';
 import type { AppProps } from 'next/app';
+import dynamic from 'next/dynamic';
 import Head from 'next/head';
 
+const Footer = dynamic(() => import('~components/Footer'));
+const ToggleColorMode = dynamic(() => import('~components/ToggleColorMode'));
```

Comment:
I don't know why this work, can you explain? I see that `ToggleColorMode` is always rendered, so in theory it shouldn't benefit from code splitting

Reply:
I'm just blindly make component to use Lazy (dynamic import), Sorry for this cause I'm NOT checking the detail.